AppController/djinn.rb

```diff
@@ -983,13 +983,15 @@ def status(secret)
         next if app_name == "none"
         stats_str << "    Information for application: #{app_name}\n"
         stats_str << "        Language            : "
-        unless @app_info_map[app_name]['language'].nil?
-          stats_str << "#{@app_info_map[app_name]['language']}\n"
-        else
+        if @app_info_map[app_name]['language'].nil?
```

Comment:
This is just reporting, and here we just report the language of the application. This is set way before the appserver is running, so technically we can see a 'pending' appserver (that is the app is not running), but we can still have the language showing properly. The case (rare) is that this call is done after the update call has been done, so the app_info_map for the app is created, but the language has not been set yet.